packages/devextreme/scss/widgets/material/popup/_index.scss

```diff
@@ -28,7 +26,7 @@ $material-popup-toolbar-label-font-size: $material-m-font-size;
 
   .dx-state-focused {
```

Comment:
I suppose that selector for focused state written incorrect. For focused state we should to move this into `& > .dx-overlay-content {` block. It's relevant for material and fluent themes. Currently, the box-shadow style will apply for all nested overlay contents.

Reply:
[c398b30042c7b20d68cf776343253260fe3d3c51]